_data/navigation.yml

```diff
@@ -30,6 +30,9 @@ t:
       - title: "Blogs"
         url: /year-archive/
         group: "year-archive"
+      - title: "Summer 2020"
```

Comment:
It's not good to add another manual on the website(there some display issue), we can add the link on the home page news section and https://servicecomb.apache.org/developers/contributing.